backend/app/src/main/java/com/moh/phlat/backend/controller/ProcessDataController.java

```diff
@@ -484,13 +497,25 @@ public ResponseEntity<ResponseMessage> plrLoad(@PathVariable Long controlTableId
 		return ResponseEntity.status(HttpStatus.OK).body(new ResponseMessage("success", 200, "PLR load process started!", controlTable));
 	}
 
-@PreAuthorize("hasAnyRole(@roleService.getAllRoles())")
-@GetMapping("/reportsummary/{controlTableId}")
-public @ResponseBody ResponseEntity<ResponseMessage> getReportSummaryByControlTableId(
-		@PathVariable Long controlTableId, @RequestParam(required = false) String rowStatus) {
-	
-	List<ReportSummary> list = processDataService.getReportSummary(controlTableId);
+	@PreAuthorize("hasAnyRole(@roleService.getAllRoles())")
+	@GetMapping("/reportsummary/{controlTableId}")
+	public @ResponseBody ResponseEntity<ResponseMessage> getReportSummaryByControlTableId(
```

Comment:
does this change belong to your story?

Reply:
It does not belong to my story.  I think it was brought in through merges from main and showed as an update.

Reply:
I am not able to understand, when you are merging the code from main into your branch, it should not show as difference here.

Reply:
I don't know what to say.  I did not make any changes to this portion of the code myself.  How it shows that I did is a mystery.  I looked at Main and the code resembles 500, 501, and 502.